test/ext-test/cairomakie/cairomakie_ext.jl

```diff
@@ -0,0 +1,63 @@
+@testset "CairoMakie Extension" verbose = true begin
+    ψ = normalize(coherent(50, 5.0) + coherent(50, -5.0))
+    xvec = yvec = -15.0:0.1:15.0
+    wig = wigner(ψ, xvec, yvec)'
```

Comment:
Is our `wigner` function output exactly same as `QuTiP`? 

Or we need a `transpose` due to the column-oriented nature of `Julia`.

How about we apply the `transpose` inside `wigner` ?

Reply:
I did a simple try, seems that our output `wigner` is same as `QuTiP`.

Just curious why are we adding this `'` (or more precisely, `transpose`, since wigner values are `Real`) here